apps/src/templates/SchoolAutocompleteDropdown.jsx

```diff
@@ -96,7 +100,30 @@ export default class SchoolAutocompleteDropdown extends Component {
     });
   };
 
+  onChange = (value) => {
+    if (value) {
+      // Cache the label for this value in case we need it for the next render.
+      this.setState({knownValue: value.value, knownLabel: value.label});
+    }
+    this.props.onChange(value);
+  };
+
   render() {
+    // value will end up either an object or a string, depending whether we have
+    // a label or not.  It appears to be the quirky behavior of react-select 1.x.
```

Comment:
Should we include the link to https://github.com/JedWatson/react-select/issues/865 here, so we can potentially notice and remove this workaround when the issue is resolved in the underlying library?